src/Pages/landing-page.jsx

```diff
@@ -0,0 +1,38 @@
+import "./pages-styles/landing-page.css";
+import { Navbar } from "../Components/Navbar/Navbar.jsx";
+import { Card, SmallCard } from "../Components/Card/Card.jsx";
+import { Hero } from "../Components/Hero/Hero.jsx";
+import { Heading } from "../Components/heading/Heading";
+
+function LandingPage() {
+    return (
+        
+        <div className="main-container">
+
+            <div className="landing-container">
+                <Hero />
+
+                <Heading name="Categories" />
+
+                <div className="landing-card-component-area">
+                    <Card collection="Trendy Collection" />
+
+                    <Card collection="Vibrant Collection" />
+
+                    <Card collection="Evergreen Collection" />
+                </div>
+
+                <Heading name="Categories" />
+
+                <div className="category-container">
+                    <SmallCard name="Men's Apparels" url={require("../assets/images/man-shirt.jpg")} />
```

Comment:
dont use require instead at top of the file do this import demoImg from "../assets/imgName.png" put this demoImg in img src attribute